common/actions/config.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import { setLanguage } from 'translations';
+import {setLanguage} from '../../common/translations/index';
```

Comment:
Also, I'd like to fix the root cause of this. `setLanguage` is working in our development environments. If it's not working in some other environment, I'd like to fix up our config so that importing `'translations'` works.